engine/src/main/resources/assets/shaders/copyDepth_vert.glsl

```diff
@@ -0,0 +1,6 @@
+
+void main() {
+    gl_Position = ftransform();
+    gl_TexCoord[0] = gl_TextureMatrix[0] * gl_MultiTexCoord0;
+    gl_FrontColor = gl_Color;
```

Comment:
I wonder if some depth information should be passed through here. I guess not. I can't check right now. =(

Reply:
Based on other codes about visualizing or playing with depth buffer that I found on the internet, this should be fine.